2021/python/1048.longest-string-train.py

```diff
@@ -0,0 +1,18 @@
+class Solution:
+    def longestStrChain(self, words: List[str]) -> int:
+        if not words or len(words) == 0:
+            return 0
+        dp = {}
+        res = 1
+        for w in sorted(words, key=len):
+            dp[w] = 1
```

Comment:
you can use defaultdict, like `dp = defaultdict(lambda _: 1)`

Reply:
Hmmmm… tried. It seems not assign `1` to default.